### DISCUSSION TITLE
Inline Comments

### DISCUSSION THREAD

**Discussion by SimonKlausLudwig** [codex-team/editor.js#1956] [Ideas]
1. Describe a problem.

Is it possible to add comments to text like its possible in CKEditor or TinyMCE ? 

See: https://www.tiny.cloud/docs/plugins/premium/comments/#

**Comment by punithbm** [comment]
Having inline comments like CKEditor will help a lot. Please let us know whether this is in your future road map. 

here: https://ckeditor.com/collaboration/comments/demo/

**Comment by neSpecc** [comment]
> Please let us know whether this is in your future road map.

We really want this feature. But we need to implement some new API layers before that. We're working on it.

**Comment by SimonKlausLudwig** [comment]
Is there anything I could help on this one ?

**Comment by ngmiduc** [comment]
That's a cool feature to have!!!!

**Comment by Lexita-LT** [comment]
+

**Comment by SimonKlausLudwig** [comment]
are there any news on this ?

**Comment by Wampfer** [comment]
Yeah, this would be really helpful!

**Comment by doroszenko** [comment]
Hi, any update?

**Comment by nathanmaru** [comment]
updates?

**Comment by swiser-gus** [comment]
I really need this one as well 😅  how can we help? 👀

**Reply by thakurvijendar** [reply]
Hi 
We need the commenting feature in editor js.

**Comment by cvollmer** [comment]
Is this on the roadmap still?

**Reply by james-kaguru** [reply]
@neSpecc .whats the progress on this and how can I help out?

**Comment by osain-az** [comment]
@james-kaguru, @cvollmer , @thakurvijendar @swiser-gus  @neSpecc,  @nathanmaru  I made an inline plugin too that allows you to add comment. 

check  out this https://github.com/osain-az/editorjs-comment
your feedback is highly appreciated

**Reply by james-kaguru** [reply]
Nice work will check it out

**Comment by osain-az** [comment]
> Nice work will check it out

Thanks,  feel free to make a suggestion or contribution
